installer/install-bash-language-server.cmd

```diff
@@ -1,5 +1,3 @@
 @echo off
 
-cd /d %~dp0
-
-call npm_install bash-language-server bash-language-server
+call "%~dp0\npm_install" bash-language-server bash-language-server

```

Comment:
This was my lazy thing. But could you please add `.cmd` for this `npm_install ?

Reply:
I added `.cmd` for all `call "%~dp0\xxx"`.